test/runtests.sh

```diff
@@ -0,0 +1,4 @@
+set -e
```

Comment:
okay, but don't we still need the shebang line...

I guess maybe sometimes `set -ex` is more explicit... and if you have a weird shell that doesn't support `-ex` as options...